src/stores/api.ts

```diff
@@ -106,6 +106,25 @@ export const defineAPIStore = defineStore('api', {
             });
             return buildings;
         },
+        vehiclesByDispatchCenter: (state): Record<number, Vehicle[]> => {
+            const dispatchCenters: Record<number, Vehicle[]> = {};
+            state.buildings.forEach(building => {
+                const dispatchId = building.leitstelle_building_id;
+
+                // do not process dispatch centers
+                if (!dispatchId) return;
```

Comment:
this won't work for buildings that aren't assigned to any dispatch center (which is possible).

Reply:
Good to know, but if the building (and in result vehicle) isn't assigned to any dispatch center, would someone expect it to be returned by in a list partitioned by dispatch center?

Reply:
I would expect the array to be available under a nullish key (`null` or `undefined`) or maybe `-1`